KarhooUISDK/CommonUI/Views/KarhooTextInputView/KarhooPhoneInputView.swift

```diff
@@ -199,6 +202,12 @@ class KarhooPhoneInputView: UIView {
         }
     }
     
+    private func focusPhoneNumberField(){
+        if focusPhoneNumber ?? false {
```

Comment:
If method's body contains only one potential runtime, we should be using `guard` statement instead of `if`